bioconda_utils/bioconda_utils-conda_build_config.yaml

```diff
@@ -29,6 +29,28 @@ bamtools:
 r_base:
   - 4.0.*
 python:
-  - 2.7.*  *_cpython
-  - 3.6.*  *_cpython
+  - 3.8.*  *_cpython
   - 3.7.*  *_cpython
+  - 3.6.*  *_cpython
+  - 2.7.*  *_cpython
+
+# conda-forge-pinning>=2020.07.13.05.23.11 uses clang=10.
```

Comment:
My initial plan was to just update to `gcc=9` ahead of `conda-forge-pinning` here.
But I got really tired of the rebuilds/recipe fixes lately and am not willing to handle the nearly 2k recipes that would be rebuild by the compiler changes right now.
Postponing that until `conda-forge-pinning` gets updates GCC itself. There's no concrete schedule for that, yet, probably this fall for `gcc=9` (GCC 9 was suggested over GCC 10 due to compatibility concerns the last time this has been discussed on the conda-forge/core meeting).